app/src/main/java/com/upversionlab/veganapp/activity/PingActivity.java

```diff
@@ -0,0 +1,58 @@
+package com.upversionlab.veganapp.activity;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.upversionlab.veganapp.R;
+import com.upversionlab.veganapp.model.Ping;
+import com.upversionlab.veganapp.model.endpoint.PingInterface;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class PingActivity extends AppCompatActivity {
+
+    // Borcat's IP
+    public static final String VEGAN_SERVER = "http://192.168.0.5:8080/";
```

Comment:
I know this is a placeholder, but we should considere how we'll do this for real.
We should probably look at some `flavors` in our build (there's more on that in the Udemy's videos).